<launch>

  <arg name="device_id" value="1"/>
  <arg name="device_num" value="2"/>
  <arg name="broadcast_ip" value="192.168.50.255"/>
  <arg name="topic_send_way" value="false"/>

  <node pkg="swarm_bridge" name="bridge_node" type="bridge_udp_node" output="screen">
 
    <param name="broadcast_ip" value="$(arg broadcast_ip)" type="string"/>
    <param name="device_id" value="$(arg device_id)"/>
    <param name="odom_max_freq" value="70"/>
    <param name="device_num" value="$(arg device_num)"/>
    <param name="topic_send_way" value="$(arg topic_send_way)"/>
    <remap from="/cloud_xyz_topic" to="/cloud_xyz_topic"/>
    <remap from="/imu_topic" to="/imu_topic"/>
    <remap from="/delta_time_topic" to="/delta_time_topic"/>

  </node>
  
  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find swarm_bridge)/rviz_config/receive_map.rviz" required="true"/>

</launch>

